src/junit.rs

```diff
@@ -84,7 +97,7 @@ fn populate(
         t.filename.as_deref(),
         network,
     );
-    t.computed_name = computed_name;
+    t.computed_name = ValidatedString::from_string(computed_name).unwrap();
```

Comment:
If we unconditionally call `unwrap` on this, why does it need to be a `ValidatedString` in the first place?
This `unwrap` here looks like something that might be hit in production.